pmoves/services/node-registry/api.py

```
@@ -0,0 +1,536 @@
+
+        # Check service state
+        is_running = self.registry._running
+
+        # Check NATS connection
+        nats_connected = self.registry._nc is not None
+        if self.registry._nc:
+            try:
+                # Try to get current status
+                nats_status = self.registry._nc.status
+                nats_connected = nats_status.connected
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)